src/stores/array_formula_highlight.ts

```diff
@@ -12,29 +13,25 @@ export class ArrayFormulaHighlight extends SpreadsheetStore {
   }
 
   get highlights(): Highlight[] {
-    const zone = this.getHighlightZone();
+    let zone: Zone | undefined;
+    const position = this.model.getters.getActivePosition();
+    const cell = this.getters.getEvaluatedCell(position);
+    const spreader = this.model.getters.getArrayFormulaSpreadingOn(position);
+    zone = spreader
+      ? this.model.getters.getSpreadZone(spreader, true)
```

Comment:
I would force to name the boolean argument
```suggestion
      ? this.model.getters.getSpreadZone(spreader, { ignoreSpillError: true })
```